packages/ddp-server/session.js

```diff
@@ -204,12 +224,47 @@ export default class Session {
     return ret;
   }
 
+  messages(subscriptionHandle, collectionName, messages) {
```

Comment:
This appears to be the only place where we need to unwind the buffered messages and apply some logic based on the actions.